ospf6d/ospf6_neighbor.c

```diff
@@ -848,16 +848,30 @@ DEFPY(ipv6_ospf6_p2xp_neigh_poll_interval,
 	return CMD_SUCCESS;
 }
 
+static const char *ospf6_neighbor_state_message(struct ospf6_neighbor *on)
+{
+	/* Neighbor State */
+	if (on->ospf6_if->type == OSPF_IFTYPE_POINTOPOINT)
+		return "PointToPoint";
+	if (on->router_id == on->drouter)
+		return "DR";
+	if (on->router_id == on->bdrouter)
+		return "BR";
+	return "DROther";
+
+}
+
```

Comment:
nit: Drop this redundant empty line, please.

Reply:
done

Reply:
done

Reply:
Not yet.